Modules/vc-module-export/VirtoCommerce.ExportModule.Tests/ExportedTypeMetadataTests.cs

```diff
@@ -0,0 +1,22 @@
+using System.Threading.Tasks;
+using Xunit;
+using VirtoCommerce.ExportModule.Core.Model;
+using System.Linq;
+
+namespace VirtoCommerce.ExportModule.Tests
+{
+    public class ExportedTypeMetadataTests
+    {
+        [Fact]
+        public async Task GetFromType()
```

Comment:
Better to name test method more descriptive, e.g. GetFromType_Price_BuiltCorrectly().
https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#naming-your-tests